packages/extension/src/components/molecules/TokenDisplay/TokenInfo/TokenInfo.tsx

```diff
@@ -25,17 +27,33 @@ export const TokenInfo = forwardRef((props: TokenInfoProps, ref: Ref<HTMLDivElem
     tokenWarningMessage,
     balance,
     issuerName,
+    issuerAddress,
     ...otherProps
   } = props;
 
-  const displayToken =
-    token.length > MAX_TOKEN_LENGTH ? `${token.slice(0, MAX_TOKEN_LENGTH)}...` : token;
+  const isLPToken = useMemo(() => {
+    return token === LP_TOKEN;
+  }, [token]);
+
+  const displayToken = useMemo(() => {
+    return token.length > MAX_TOKEN_LENGTH && !isLPToken
+      ? `${token.slice(0, MAX_TOKEN_LENGTH)}...`
+      : token;
+  }, [token, isLPToken]);
+
+  const formattedIssuerAddress = useMemo(() => {
+    return issuerAddress && issuerAddress.length > 20
```

Comment:
Is this possible that the address don't have the same size?

Reply:
yes